pkg/apis/eventing/v1beta1/eventtype_defaults_test.go

```diff
@@ -35,9 +35,7 @@ func TestEventTypeDefaults(t *testing.T) {
 		"nil spec": {
 			initial: EventType{},
 			expected: EventType{
-				Spec: EventTypeSpec{
-					Broker: "default",
-				},
+				Spec: EventTypeSpec{},
 			},
 		},
 		"broker empty": {
```

Comment:
I find the name of the test confusing, w/ the new behavior. Perhaps `default broker reference`, see https://github.com/knative/eventing/pull/8030/files